src/components/BookCard/BookCard.tsx

```diff
@@ -7,7 +11,22 @@ import type { Book } from '../../domain/books/types'
 type BookProps = Book;
 
 export const BookCard = (props: BookProps): JSX.Element => {
+  const dispatch: AppDispatch = useDispatch();
+
```

Comment:
Mostly if not always try to destructure component props.
`({name, author, description, price}: BookProps)`

Reply:
Fixed